medical_medicament/tests/test_medical_drug_route.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Dave Lasley <dave@laslabs.com>
+#    Copyright: 2016-TODAY LasLabs, Inc. [https://laslabs.com]
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.tests.common import TransactionCase
+from psycopg2 import IntegrityError
+
+
+class TestMedicalDrugRoute(TransactionCase):
+
+    def setUp(self, ):
```

Comment:
Cool yeah I'll run a regex to knock them out as we work through the PRs. 

My IDE adds the `self, ` and I use tab to navigate directly to the method contents, which bypasses the args entirely. 

For the most part I just add them by habit on the other args because it doesn't actually hurt anything IMO. Seems that it bugs everyone else, so I'll be more diligent with it :wink: